TAC-Requirements.md

```diff
@@ -0,0 +1,52 @@
+# Requirements to join ACT
+The following document outlines requirements for projects interested in joining the Automating Compliance Tooling Project umbrella. All ACT umbrella projects are required to participate in bi-monthly Technical Advisory Committee (TAC) calls. ACT projects and affiliate projects maintain one vote in all TAC voting body decisions.
+
+## General Criteria for new projects to join the ACT umbrella
+* Willingness to generate/consume/interoperate with [SPDX](https://spdx.dev/).
+* Formal governance in place meeting Linux Foundation governance requirements.
```

Comment:
@kestewart is there a reference link available for this?

Reply:
How about rephrasing this as:
* Formal project technical charter in place approved by the Linux Foundation.   

There is no real "one size" I can point to.